integration/helpers.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2022 The Inspektor Gadget authors
```

Comment:
I wonder if naming the files with `_test.go` prefix is a better idea like `command_test.go/helpers_test.go`. WDYT?

Reply:
Mmmm, I noticed you did that on https://github.com/kinvolk/inspektor-gadget/pull/946. I am not sure, using `_test.go` makes sense because they should be only considered by `go test`. However, they don't include tests at all, so it doesn't look natural to use the `_test.go` convention.

Reply:
Make sense. I will follow the same convention in #946 then!